create-custom-map-style/tileserver/startServer.sh

```diff
@@ -0,0 +1,5 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+#!/bin/bash
+
+npx tessera -r $(pwd)/node_modules/tilelive-aws/tilelive-aws.js aws:///CreateCustomMapStyle
```

Comment:
See if just `-r tilelive-aws` works

Reply:
Again my shell command skills aren't the best, but I couldn't get it to work any other way than this. I'm using iTerm 2 with zsh so might have something to do with that.

Reply:
Nah, I think you're right. Could be an `npx` thing...

Actually, since you're including `tilelive-aws` as a dependency, it wouldn't hurt to also add `tessera` (vs. having it downloaded each time using `npx`). This would change to `node_modules/.bin/tessera ...` instead.